xpti/include/xpti/xpti_trace_framework.hpp

```diff
@@ -663,6 +1589,7 @@ class tracepoint_t {
         m_trace_event = xptiMakeEvent(
             m_default_name, const_cast<xpti::payload_t *>(m_payload),
             m_default_event_type, m_default_activity_type, &m_instID);
+      } else {
```

Comment:
why do we need this?

Reply:
For backward compatibility tests and in the next pass, it will be deprecated.